src/speakers.js

```diff
@@ -46,6 +46,7 @@ export default () => (
 
             <Speaker author="Stephen Diehl" img={require('../static/stephen-diehl.jpg')} github='sdiehl' twitter='smdiehl'>
                 Software engineer in Boston, Massachusetts.
+                Author of <a href="http://dev.stephendiehl.com/hask/">What I Wish I Knew When Learning Haskell</a>.
```

Comment:
FYI, I asked the speakers to provide a short descriptions for themselves. Already received one from Neil.